src/components/fruitThinks.jsx

```diff
@@ -0,0 +1,38 @@
+
+import "./fruitThinks.css"
+import React from "react";
+
+function FruitThinks({fruitImg, fruitName, setValue, value}) {
+
+
+    return (
+        <>
+        <article>
+            <div className="fruitTinksTitle">
+                <h1>{fruitImg}</h1>
+                <h1>{fruitName}</h1>
+            </div>
+
+            <div className="fruitTinksButton">
+            <button
+                type="button"
+                onClick={() => value > 0 && setValue(value - 1)}
```

Comment:
De `value > 0` check zou ik als expressie meegeven aan de `disabled` property. Dan wordt de onClick niet eens uitgevoerd 😉. Dat is namelijk de juiste manier om dit gedrag te krijgen (en kun je ook veel makkelijker qua css een andere styling geven als het disabled is, omdat je daar geen toegang hebt tot `value`, maar wél weet of een button disabled is of niet)